osu.Server.Queues.ScoreStatisticsProcessor/Commands/Maintenance/DeleteNonPreservedScoresCommand.cs

```diff
@@ -33,7 +37,7 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                 await deleteCommand.PrepareAsync(cancellationToken);
 
                 var scores = await readConnection.QueryAsync<SoloScore>(new CommandDefinition(
-                    $"SELECT * FROM scores WHERE preserve = 0 AND updated_at < DATE_SUB(NOW(), INTERVAL {preserve_hours} HOUR)", flags: CommandFlags.None, cancellationToken: cancellationToken));
+                    $"SELECT * FROM `scores` WHERE `preserve` = 0 AND `unix_updated_at` < UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL {preserve_hours} HOUR))", flags: CommandFlags.None, cancellationToken: cancellationToken));
```

Comment:
Please scrutinise this change because the query used the old score structure and as such is currently non-functional against latest master of web.

Reply:
looks correct 👍